README.md

```diff
@@ -88,6 +88,7 @@ The following options must be set:
   - `analysisProcess.service` - The service to run the analysis on. Either `docker` (for local development and off-line) or `hyper`. [ANL_SERVICE]
   - `analysisProcess.hyperAccess` - Access key for Hyper. [HYPER_ACCESS]
   - `analysisProcess.hyperSecret` - Secret key for Hyper. [HYPER_SECRET]
+  - `analysisProcess.hyperSize` - The size of the Hyper container. If not specified, it will use Hyper's default container. [HYPER_SIZE]
```

Comment:
Should there be a link to the hyper pricing page to know what sizes are available?